source/manuals/monitor-paas-app-with-prometheus.html.md.erb

```diff
@@ -94,12 +92,18 @@ To set up the metrics exporter app:
 
 4. Run `cf start metric-exporter` to start your app
 1. Check you're generating Prometheus metrics at the metrics endpoint
-    - `https://<your-app-url>/metrics`
+    - `https://<app-name>.cloudapps.digital/metrics`
 5. Bind your app to the Prometheus service
-  - `cf bind-service <app-name> <service-instance-name>`
+  - `cf bind-service metric-exporter <service-instance-name>`
+6. Map a route to your app so Prometheus can scrape it
+  - `cf map-route metric-exporter cloudapps.digital --hostname <app-name>`
```

Comment:
these steps are in the wrong order - step one is "check the route is working", but step 6 is "set up the route".  We need to set the route before it'll work!